osc_sdk_python/authentication.py

```diff
@@ -15,6 +16,8 @@ def __init__(self, credentials, host,
                  user_agent = DEFAULT_USER_AGENT):
         self.access_key = credentials.get_ak()
         self.secret_key = credentials.get_sk()
+        self.email = credentials.get_email()
```

Comment:
TBH I'm not a big fan of those getter.
I would prefer if we use `credentials.email`, and so on.